packages/eclipse/src/components/ui/tabs.tsx

```
@@ -59,12 +59,14 @@ export function Tabs({
 }: TabsProps) {
   const tabsRef = useRef<HTMLDivElement>(null);
   const valueToIdMap = useMemo(() => new Map<string, string>(), []);
+  const onValueChangeRef = useRef(_onValueChange);
+  onValueChangeRef.current = _onValueChange;
   const [value, setValue] =
     _value === undefined
       ? // eslint-disable-next-line react-hooks/rules-of-hooks -- not supposed to change controlled/uncontrolled
         useState(defaultValue)
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)